python/pyarrow/filesystem.py

```diff
@@ -135,6 +135,12 @@ def isfile(self, path):
         """
         raise NotImplementedError
 
+    def isfilestore(self):
```

Comment:
Can you make this a private API (`_isfilestore`)? Unclear if normal users would need this

Reply:
👍  updated